packages/jest-config/src/getMaxWorkers.js

```diff
@@ -12,7 +12,7 @@ import type {Argv} from 'types/Argv';
 import os from 'os';
 
 export default function getMaxWorkers(argv: Argv): number {
-  if (argv.runInBand) {
```

Comment:
I think it makes more sense to put the check before we do `.length`. In this case, we want `maxWorkers` to override the fallback of 1

Reply:
Sure, makes sense. I'll push an updated version.